packages/react-composites/src/composites/CallComposite/ScreenSharePopup.tsx

```diff
@@ -69,7 +69,7 @@ export const ScreenSharePopup = (props: ScreenSharePopupProps): JSX.Element => {
   }, [onStopScreenShare]);
 
   // Beware: The FocusTrapCallout is a workaround to allow users to tab-navigate out of the Modal. There is an open
-  // issue here: https://github.com/microsoft/fluentui/issues/16179
+  // issue here: https://github.com/microsoft/fluentui/issues/18924
```

Comment:
This issue is not resolved but stayed too long in fluentUI repo, reopened a new issue and hopefully we will raise some additional awareness from fluentui team
(There should be some better work around when we wanna make some accessibility features, like adding onBlur behavior of the stop button)